packages/react-composites/tests/browser/call/TestCallingState.ts

```diff
@@ -19,3 +20,39 @@ export type TestRemoteParticipant = {
   isVideoStreamAvailable?: boolean;
   isScreenSharing?: boolean;
 };
+
```

Comment:
Yes, unfortunately. I cannot seem to import modules from our Typescript modules because in playwright automatically transforms TypeScript to JavaScript and they mention it may not transpile correctly. 
Fortunately, if there are any breaking changes in the CallingSDK types then the test browser app will show errors since the `TestCallingState` is used to form the `CallAdapterState` in `MockCallAdapter`